pkg/cache/cache.go

```diff
@@ -36,6 +38,9 @@ type SharedInformerCache interface {
 
 	// ListTransformedPods lists the all the Pods from the cache after applying TransformFunc
 	ListTransformedPods() (*corev1.PodList, error)
+
+	// WatchPods watches registers the pod event handler with the informer cache
```

Comment:
nit..watches registers?
I think need to remove watches or need to reword it

Reply:
modified